src/bufferedscanlines.c

```diff
@@ -28,118 +28,117 @@
 /*
  * private routine to clear a buffer and reset the currentRow
  */
-static void bufferscan_reset(t_bufferscan *bs)
-{
-	if (!bs) return;
-	int totalBytes;
-
-	if (!bs) return;
-	totalBytes = bs->bytesPerRow * bs->currentRow;
-	bs->currentRow = 0;
-	memset(bs->rawData, 0, totalBytes);
+static void bufferscan_reset(t_bufferscan *bs) {
```

Comment:
Most of these changes are whitespace changes and don't add or fix anything in the existing code.
I don't know if this was intentional or not, but perhaps you'd like to limit your PR to material changes in the code base. Perhaps this was done by a pretty printing editor and you weren't aware? Consider excluding this file from the PR.